django_airavata/apps/resourceallocation/views.py

```diff
@@ -0,0 +1,280 @@
+
+        details = request.allocation_manager_client.getAllRequestsForAdmin(authz_token,loggedinUser)
+        reviewerDetailsList = []
+        for detail in details:
+            assignedReviewerDetails = request.allocation_manager_client.getAllAssignedReviewersForRequest(authz_token,
+                detail.projectId)
+            allAssignedReviewerList= []
+            for reviewer in assignedReviewerDetails:
+                allAssignedReviewerList.append(reviewer.reviewerUsername)
+            unassignedReviewersList = list(set(allReviewersList) - set(allAssignedReviewerList))
+            reviewerDetailsList.append(unassignedReviewersList)
+        return render(request, 'dashboard/admin.html',
+                      {'all_requests': details, 'reviewerDetailsList': reviewerDetailsList})
+
+    except Exception as e:
+        logger.exception("Failed to load resource allocation details")
+        return redirect('/resourceallocation')
+
+@login_required
+def AdminRequestView(request):
+    authz_token = request.authz_token
+    loggedinUser = authz_token.claimsMap['userName']
+    projectId = int(request.GET.get('projectId'))
+    projectReviewList = request.allocation_manager_client.getAllReviewsForARequest(authz_token,projectId)
+    reviewerReview = ReviewerAllocationDetail()
+    userSubmittedDetails = request.allocation_manager_client.getAllocationRequest(authz_token,projectId)
+    userSpecificDetailsList = request.allocation_manager_client.getUserSpecificResource(authz_token, projectId)
+    specificDetails = request.allocation_manager_client.getReviewerSpecificResource(authz_token, projectId)
+    reviewerSpecificDetailsList = []
+    selectedReviewer = ''
+
+    if (request.method == 'POST'):
+        s = request.POST.get('start')
+        rejectionReason = request.POST.get('reject-reason')
+        specificResource = request.POST.get('specific-resource')
+        startDate = None
+        if s is not None:
+            startDate = time.mktime(datetime.datetime.strptime(s, "%Y-%m-%d").timetuple())
+            s = request.POST.get('end')
+            endDate = time.mktime(datetime.datetime.strptime(s, "%Y-%m-%d").timetuple())
+            serviceUnits = request.POST.get('allocation-units')
+
+        if(startDate is not None):
+            request.allocation_manager_client.approveRequest(authz_token,projectId,loggedinUser,int(startDate),int(endDate),int(serviceUnits),specificResource)
+            return redirect('/resourceallocation/admin')
+        if rejectionReason is not None:
+            request.allocation_manager_client.rejectRequest(authz_token,projectId,loggedinUser,rejectionReason,specificResource)
+            return redirect('/resourceallocation/admin')
+        selectedReviewer = request.POST.get('selectedReviewer')
+        for review in projectReviewList:
+            if(review.username == selectedReviewer):
+                reviewerReview = review
+        print(specificDetails)
```

Comment:
Please use a logger instead of print statements. To do so, at the top of the file, after your imports create a logger
```
logger = logging.getLogger(__name__)
```

Then you can log like so
```
logger.debug(specificDetails)
```